utils/build_swift/driver_arguments.py

```diff
@@ -713,99 +735,73 @@ def create_argument_parser():
-        '--cmark-assertions',
-        action='store_const',
-        const=True,
-        dest='cmark_assertions',
-        help='enable assertions in CommonMark')
-    assertions_override_group.add_argument(
-        '--llvm-assertions',
-        action='store_const',
-        const=True,
-        dest='llvm_assertions',
-        help='enable assertions in LLVM')
-    assertions_override_group.add_argument(
-        '--no-llvm-assertions',
-        action='store_const',
-        const=False,
-        dest='llvm_assertions',
-        help='disable assertions in LLVM')
-    assertions_override_group.add_argument(
-        '--swift-assertions',
-        action='store_const',
-        const=True,
-        dest='swift_assertions',
-        help='enable assertions in Swift')
-    assertions_override_group.add_argument(
-        '--no-swift-assertions',
-        action='store_const',
-        const=False,
-        dest='swift_assertions',
-        help='disable assertions in Swift')
-    assertions_override_group.add_argument(
-        '--swift-stdlib-assertions',
-        action='store_const',
-        const=True,
-        dest='swift_stdlib_assertions',
-        help='enable assertions in the Swift standard library')
-    assertions_override_group.add_argument(
-        '--no-swift-stdlib-assertions',
-        action='store_const',
-        const=False,
-        dest='swift_stdlib_assertions',
-        help='disable assertions in the Swift standard library')
-    assertions_override_group.add_argument(
-        '--lldb-assertions',
-        action='store_const',
-        const=True,
-        dest='lldb_assertions',
-        help='enable assertions in LLDB')
-    assertions_override_group.add_argument(
-        '--no-lldb-assertions',
-        action='store_const',
-        const=False,
-        dest='lldb_assertions',
-        help='disable assertions in LLDB')
-
-    # FIXME: This should be one option using choices=[...]
-    cmake_generator_group = parser.add_argument_group(
-        title='Select the CMake generator')
-    cmake_generator_group.add_argument(
-        '-x', '--xcode',
-        action='store_const',
-        const='Xcode',
-        dest='cmake_generator',
-        help="use CMake's Xcode generator (default is Ninja)")
-    cmake_generator_group.add_argument(
-        '-m', '--make',
-        action='store_const',
-        const='Unix Makefiles',
-        dest='cmake_generator',
-        help="use CMake's Makefile generator (default is Ninja)")
-    cmake_generator_group.add_argument(
-        '-e', '--eclipse',
-        action='store_const',
-        const='Eclipse CDT4 - Ninja',
-        dest='cmake_generator',
-        help="use CMake's Eclipse generator (default is Ninja)")
-
+    # -------------------------------------------------------------------------
+    # Assertions group
+
+    with mutually_exclusive_group():
+        set_defaults(assertions=True)
+
+        # TODO: Convert to store_true
```

Comment:
Why do we need to use "store_true" here?

Reply:
Because using `store_const` and then setting the const to `True` is a bit ridiculous, since that's almost `store_true`, just without the safety of setting the default to `False`. It doesn't make sense to store a constant boolean for an option, just use `store_true`, `store_false`, `toggle_true` and `toggle_false` instead.